server/controller/itemController.ts

```diff
@@ -0,0 +1,53 @@
+import { Request,Response} from 'express';
+import ItemService from '../service/itemService';
+
+class ItemController {
+
+async create(req : Request  , res : Response ){
+try {
+const record = await ItemService.createItems(req);
+console.log(record);
+return res.json({record, msg : "Item has been created successfully"});
+}
+catch {
+  return res.json({msg: "fail to create" , status: 500, route: "/api/jiraBoardData"});
+}
+}
+
+
+  async read(req : Request , res : Response){
+  try{
+    const record = await ItemService.readItems();
```

Comment:
If `readItems` method returns array, can we replace record with its plural version, or other clearer name